src/js/pinny.js

```diff
@@ -148,7 +156,15 @@
                         $window.off(events.orientationchange);
                     }
 
-                    this.$container.attr('aria-hidden', 'false');
+                    // Lockup a11y fix
+                    // Instead of making whole lockup container aria hidden,
+                    // just make page content aria hidden, since otherwise pinny
+                    // cannot be read by screenreaders
+                    if ($('#x-root').length) {
+                        $('#x-root').attr('aria-hidden', 'false');
```

Comment:
This string is being used multiple times. Should probably make this a constant? Perhaps as part of `classes`.

Also, can this be made configurable? I feel like there may be some cases in which a different container will need to be targeted.